datasets/CodeCompliantEquipment.idf

```diff
@@ -0,0 +1,2427 @@
+!---------------------------------------------------------------------------------------------
+!- This dataset includes performance curve objects that describe the part load performance of
+!- building energy code minimally compliant equipment.
+!-
+!- The dataset provides curves for the following EnergyPlus object:
+!-   * Chiller:Electric:EIR
+!-
+!- Sets of curves are provided for the following building energy codes minimum requirements:
+!-   * ASHRAE Standard 90.1-2019 Table 6.8.1-3
+!-   * ASHRAE Standard 90.1-2019 Table G3.5.3
+!-
+!- Last update: Sept. 2021
+!---------------------------------------------------------------------------------------------
+
+!----------------------------------------------------------
+!- Beginning of performance curves for Chiller:Electric:EIR:
+
+Curve:Quadratic,
+    ASHRAE901_AppG_wtr_cent_gte150lt300ton_0.634kWpton_0.596IPLV.IP_eir-f-plr,    !- Name
```

Comment:
Not a huge problem, but it appears that all of the eir-f-plr curves are at the top, then the other curves are grouped by chiller size etc.